Drivers/STM32F4xx_HAL_Driver/Inc/stm32f4xx_hal_i2c.h

```diff
@@ -0,0 +1,740 @@
+/**
```

Comment:
We gotta double check and make sure this is the correct way of including I2C stuff in STM32 repos